instructure/instructure-ui
# chore: add cypress component testing

INSTUI-3958

test:

- run component tests locally in headless mode
- run component tests in non-headless mode
- check [documentation](https://1402--preview-instui.netlify.app/#integration-testing)
